### ISSUE TITLE
Router interface becomes extremely slow

### ISSUE THREAD

**Issue by paull27** [louis49/homebridge-plugin-androidtv#46]
Whenever the plugin is active, the web interface of my router (Fritzbox 7560) becomes extremely slow. Even if I only connect a single adb device and set the polling interval to 10000 (max value). 
As soon as I deactivate the plugin (or child bridge) it works perfectly.

Do you have any idea for solving this issue?

**Comment by louis49** [comment]
Yeah, I think the problem is that a send some broadcast requests : I need to limit this only when user is seeking to pair devices
